modules/imgcodecs/include/opencv2/imgcodecs.hpp

```diff
@@ -85,22 +85,24 @@ enum ImwriteFlags {
        IMWRITE_WEBP_QUALITY        = 64  //!< For WEBP, it can be a quality from 1 to 100 (the higher is the better). By default (without any parameter) and for quality above 100 the lossless compression is used.
      };
 
-//! Imwrite PNG specific flags
+//! Imwrite PNG specific flags used to tune the compression algorithm.
```

Comment:
I'd add an additional note, something like: `These flags will be modify the way of PNG image compression and will be passed to the underlying zlib processing stage.`